sdk/iot/provisioning/readme.md

```diff
@@ -0,0 +1,54 @@
+# Azure IoT Clients for Embedded C: Azure IoT Device Provisioning Client
+
+Official Embedded C client for Azure IoT DPS - Device Provisioning Services.
+
+For more details about DPS check [Azure documentation](https://docs.microsoft.com/azure/iot-dps/)
+
+## Getting started
+
+- Pelase refer to the state machine diagram which describes the Azure SDK for Embedded C flow for DPS [here](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/iot/doc/mqtt_state_machine.md#device-provisioning-service). 
```

Comment:
Avoid using DPS...long history. You can use "Provisioning Service"

Reply:
fixed.